Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -169,6 +175,10 @@ extension ModuleError: CustomStringConvertible {
             by activating the automatic source-control to registry replacement, or by using mirrors. \
             if they are not duplicate consider using the `moduleAliases` parameter in manifest to provide unique names
             """
+        case .invalidTrait(let package, let trait):
+            return """
+            Tried to enable invalid trait "\(trait)" for package \(package)
```

Comment:
IMO this error message is not actionable. Can you give it more clarity on what "invalid" means in this context and how a user would be able to address it?